CMakeLists.txt

```diff
@@ -40,6 +40,20 @@ int main(int argc, char *argv[]) {
   if(DISPATCH_HAVE_EXTENDED_SLPI_20348)
     add_compile_definitions(DISPATCH_HAVE_EXTENDED_SLPI_20348)
   endif()
+
```

Comment:
not sure what's up with the github diff viewer but this diff isn't real, this file is identical between the two branches